example/lib/star_wars/hooks.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:flutter_hooks_gallery/star_wars/redux.dart';
+import 'package:flutter_hooks_gallery/star_wars/star_wars_api.dart';
+import 'package:provider/provider.dart';
+
+/// fetch planets and update store
```

Comment:
This comment doesn't bring any extra information that the name doesn't already tell

It might be interesting to explain what the parameter is, add a  "See also", and explain how it's supposed to update the store